src/Symfony/Component/Security/composer.json

```diff
@@ -39,11 +39,6 @@
         "ircmaxell/password-compat": "~1.0",
         "symfony/expression-language": "~2.6"
     },
-    "replace": {
-      "symfony/security-acl": "self.version",
```

Comment:
actually, only this entry should be removed. Core and HTTP should still be replaced by the current version (as it's included in the Security package).